timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -279,6 +295,52 @@ def _get_scale_per_item(self, item_ids: pd.Index) -> pd.Series:
         else:
             return pd.Series(1.0, index=item_ids)
 
+    def _generate_fallback_forecast_for_short_series(
+        self,
+        data: TimeSeriesDataFrame,
+        known_covariates: Optional[TimeSeriesDataFrame] = None,
+    ):
+        """Generate naive forecast for time series that are too short for chosen differencing."""
+        ts_lengths = data.num_timesteps_per_item()
+        short_series = ts_lengths.index[ts_lengths <= self._sum_of_differences]
+        if len(short_series) > 0:
+            logger.warning(
+                f"Warning: {len(short_series)} time series ({len(short_series) / len(ts_lengths):.1%}) are shorter "
+                f"than {self._sum_of_differences} and cannot be predicted by {self.name}. "
+                "Fallback model SeasonalNaive is used for these time series."
+            )
+            data_short = data.query("item_id in @short_series")
+            seasonal_naive = SeasonalNaiveModel(freq=self.freq, prediction_length=self.prediction_length)
+            seasonal_naive.fit(train_data=data_short)
+            forecast_for_short_series = seasonal_naive.predict(data_short)
+
+            data_long = data.query("item_id not in @short_series")
+            if known_covariates is not None:
+                known_covariates_long = known_covariates.query("item_id not in @short_series")
+            else:
+                known_covariates_long = None
+        else:
+            data_long = data
+            known_covariates_long = known_covariates
+            forecast_for_short_series = None
+        return data_long, known_covariates_long, forecast_for_short_series
+
+    def _add_gaussian_quantiles(self, predictions: pd.DataFrame, repeated_item_ids: pd.Series):
+        """
+        Add quantile levels assuming that residuals follow normal distribution
+        """
+        from scipy.stats import norm
+
+        scale_per_item = self._get_scale_per_item(repeated_item_ids.unique())
+        num_items = int(len(predictions) / self.prediction_length)
+        sqrt_h = np.sqrt(np.arange(1, self.prediction_length + 1))
+        # Series where normal_scale_per_timestep.loc[item_id].loc[N] = sqrt(1 + N) for N in range(prediction_length)
+        normal_scale_per_timestep = pd.Series(np.tile(sqrt_h, num_items), index=repeated_item_ids)
+        std_per_timestep = self._avg_residuals_std * scale_per_item * normal_scale_per_timestep
```

Comment:
Just saw that the default value for `self._avg_residuals_std` is 1.0. Why? Looks to me like this variable is better off without a default?

Reply:
also, just to cross check, this value is computed on one-step residuals right?

Reply:
Good point, set it to None (we should receive an error if model is not fit).

Indeed, it's computed on one-step-ahead forecast residuals.